R/get.R

```diff
@@ -629,6 +629,58 @@ get_oauth_credentials <- function(connect, user_session_token) {
   )
 }
 
+#' Perform an OAuth credential exchange to obtain a content-specific OAuth
+#' access token.
+#'
+#' @param connect A Connect R6 object.
+#' @param content_session_token The content session token. This token can only
```

Comment:
```suggestion
#' @param content_session_token Optional. The content session token. This token can only
```

To go along with suggestion below. And I'd also adjust the prose a little too maybe :)